genesis/src/genesis_accounts.rs

```diff
@@ -23,427 +23,111 @@ const UNLOCKS_BY_TENTHS_FOR_60_MONTHS: UnlockInfo = UnlockInfo {
-        sol: 2_579_350.0,
-    },
-    StakerInfo {
-        name: "dramatic veil",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 3_611_110.50,
-    },
-    StakerInfo {
-        name: "incandescent skin",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 3_000_000.0,
-    },
-    StakerInfo {
-        name: "spiky love",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 3_250_000.0,
-    },
-];
-
-const BATCH_TWO_STAKER_INFOS: &[StakerInfo] = &[
-    // 2nd batch
-    StakerInfo {
-        name: "macabre note",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
-    },
-    StakerInfo {
-        name: "alcoholic letter",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
-    },
-    StakerInfo {
-        name: "heady trucks",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
-    },
-    StakerInfo {
-        name: "ten support",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 1_000_000.0,
-    },
-    StakerInfo {
-        name: "foregoing middle",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 800_000.0,
-    },
-    StakerInfo {
-        name: "ludicrous destruction",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
-    },
-    StakerInfo {
-        name: "numberless wheel",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
-    },
-    StakerInfo {
-        name: "short powder",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
-    },
-    StakerInfo {
-        name: "cut name",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
-    },
-    StakerInfo {
-        name: "six fly",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 4_000_000.0,
+        name: "wretched texture",
+        staker: "HbENu65qjWLEB5TrMouSSWLq9mbtGx2bvfhPjk2FpYek",
+        sol: 225_000.0,
     },
     StakerInfo {
-        name: "mindless pickle",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 100_000.0,
+        name: "nutritious examination",
+        staker: "C9CfFpmLDsQsz6wt7MrrZquNB5oS4QkpJkmDAiboVEZZ",
+        sol: 5_000_000.0,
     },
     StakerInfo {
-        name: "marked rabbit",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 38_741.36,
+        name: "tidy impression",
+        staker: "6ne6Rbag4FAnop1KNgVdM1SEHnJEysHSWyqvRpFrzaig",
+        sol: 5_000_000.0,
     },
     StakerInfo {
-        name: "jagged doctor",
-        staker: "P1aceHo1derPubkey11111111111111111111111111",
-        sol: 711_258.64,
+        name: "unbecoming silver",
+        // TODO: staker: "42yapY7Vrs5jqht9TCKsPoyb4vDFYcPfRkqAP85NSAQ", WrongSize
```

Comment:
Can we track this TODO in the SSOT doc instead of here?

Reply:
Yeah, I added a comment to that cell and will follow up with Jeff as well